src/libraries/System.Security.Cryptography.ProtectedData/tests/ArrayProtectedDataTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace System.Security.Cryptography.ProtectedDataTests;
```

Comment:
I don't think we typically use file-scoped namespaces in System.Security tests.

Reply:
You're right, sorry. The IDE did it for me. I'll block scope them all.